<launch>
    <arg name="sequence" default="00" />
    <node name="hvlo" pkg="hybrid_vlo" type="hvlo" output="screen">
        <param name="strSettingPath" type="string" value="$(find hybrid_vlo)/config/setting_file.yaml"/>
        <param name="strTrajectorySavePath" type="string" value="$(find hybrid_vlo)/results/vlo_path_$(arg sequence).txt"/>
    </node>
    <node name="hybrid_vlo_rviz" pkg="rviz" type="rviz"  args="-d $(find hybrid_vlo)/config/hybrid_vlo.rviz" output="screen"/>
</launch>